08-fraction_calculator/08-fraction_calculator.java

```diff
@@ -0,0 +1,75 @@
+import java.util.Scanner;
+
+class Main {
+  public static void main(String[] args) {
+    Scanner in = new Scanner(System.in);
+
+    boolean quit = true;
+
+    do { //i know there is a different type of while loop but i know how to use this one
```

Comment:
This should definitely be fine! I guess we would just ask whenever you use something that may be "different" go a bit more into depth in your comments about how it works and how you are using. It will help later on for clarity, aside from that, looking good! ^^